src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,92 @@
+package core.basesyntax.service;
+
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static RegistrationServiceImpl registrationService;
+    private static StorageDaoImpl storageDao;
+    private static User user;
+
+    @BeforeAll
+    static void beforeAll() {
+        registrationService
+                = new RegistrationServiceImpl();
+        storageDao = new StorageDaoImpl();
+        user = new User();
+    }
+
+    @BeforeEach
+    void setUp() {
+        user.setAge(18);
+        user.setLogin("user'sLogin");
+        user.setPassword("user'sPassword");
+    }
```

Comment:
Whet will happen if one of your future test will try to add such user? 

        user.setAge(18);
        user.setLogin("user'sLogin");
        user.setPassword("user'sPassword");